editor/testing.md

```diff
@@ -158,6 +159,9 @@ refresh materialized view location_crashes_view;
 - The **Select person** button is displayed for each EMS patient row
 - Click **Select person** to enable the **Select match** button to appear next to any unlinked person records in the **Associated people records** table
 - Click the **Person ID** column for any **EMS Patients** row to manually edit a person ID value
+- - Click the **Person ID** column submit an invalid person ID value and verify an error message is displayed
+- Use the falafel menu to **Reset** an incident matched to a person ID
+- Use the falafel menu to modify an incident to be **Match not found**
```

Comment:
i updated the test doc on the wrong branch. this commit is already in main via cherry-pick